pkg/build/builder/dockerutil.go

```diff
@@ -110,8 +110,15 @@ func pullImage(client DockerClient, name string, authConfig docker.AuthConfigura
 	logProgress := func(s string) {
 		glog.V(0).Infof("%s", s)
 	}
+	tag := "latest"
```

Comment:
eventually reference.ParseNamedDockerImageReference(spec) which is what ParseDockerImageReference use internally. 

Or `github.com/docker/distribution/reference.ParseNamed()`